sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -385,8 +385,12 @@ case class LoadDataCommand(
         val hadoopConf = sparkSession.sessionState.newHadoopConf()
         val srcPath = new Path(hdfsUri)
         val fs = srcPath.getFileSystem(hadoopConf)
-        if (!fs.exists(srcPath)) {
-          throw new AnalysisException(s"LOAD DATA input path does not exist: $path")
+        // A validaton logic is been added for non local files, Error will be thrown
+        // If hdfs path doest not exist or if no files matches the wild card defined
+        // in load path
```

Comment:
Check if the path exists or there is matched path. For HDFS path, we support wildcard in directory name and file name.

Reply:
yeah i updated the comment, the above comment is more precise.